package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Sorry for the big changes, this is the only file that needs review the rest is `packge.json` the reset of the files are the result of regenerating the docs.